src/pages/ReimbursementAccount/BeneficialOwnerInfo/BeneficialOwnerInfo.js

```diff
@@ -0,0 +1,302 @@
+            return;
+        }
+
+        // User reached the limit of UBOs
+        if (currentUBOSubstep === SUBSTEP.UBO_DETAILS_FORM && !canAddMoreUBOS) {
+            setCurrentUBOSubstep(SUBSTEP.UBOS_LIST);
+        }
+    };
+
+    const handleBackButtonPress = () => {
+        // User goes back to previous step
+        if (currentUBOSubstep === SUBSTEP.IS_USER_UBO) {
+            onBackButtonPress();
+            // User reached limit of UBOs and goes back to initial question about additional UBOs
+        } else if (currentUBOSubstep === SUBSTEP.UBOS_LIST && !canAddMoreUBOS) {
+            setCurrentUBOSubstep(SUBSTEP.IS_ANYONE_ELSE_UBO);
+            // User goes back to last radio button
+        } else if (currentUBOSubstep === SUBSTEP.UBOS_LIST && isAnyoneElseUBO === true) {
+            setCurrentUBOSubstep(SUBSTEP.ARE_THERE_MORE_UBOS);
+        } else if (currentUBOSubstep === SUBSTEP.UBOS_LIST && isUserUBO === true && isAnyoneElseUBO === false) {
+            setCurrentUBOSubstep(SUBSTEP.IS_ANYONE_ELSE_UBO);
+            // User moves between substeps of beneficial owner details form
+        } else if (currentUBOSubstep === SUBSTEP.UBO_DETAILS_FORM && screenIndex > 0) {
+            prevScreen();
+        } else {
+            setCurrentUBOSubstep((currentSubstep) => currentSubstep - 1);
+        }
+    };
+
+    const handleUBOEdit = (beneficialOwnerID) => {
+        setBeneficialOwnerBeingModifiedID(beneficialOwnerID);
+        setIsEditingCreatedBeneficialOwner(true);
+        setCurrentUBOSubstep(SUBSTEP.UBO_DETAILS_FORM);
+    };
+
+    return (
+        <ScreenWrapper
+            testID={BeneficialOwnerInfo.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldEnableMaxHeight
+        >
+            <HeaderWithBackButton
+                title={translate('beneficialOwnerInfoStep.companyOwner')}
+                onBackButtonPress={handleBackButtonPress}
+                onCloseButtonPress={onCloseButtonPress}
+                shouldShowCloseButton
+            />
+            <View style={[styles.ph5, styles.mv3, {height: CONST.BANK_ACCOUNT.STEPS_HEADER_HEIGHT}]}>
+                <InteractiveStepSubHeader
+                    startStep={4}
+                    stepNames={CONST.BANK_ACCOUNT.STEP_NAMES}
+                />
+            </View>
+
+            {currentUBOSubstep === SUBSTEP.IS_USER_UBO && (
+                <BeneficialOwnerCheckUBO
+                    title={`${translate('beneficialOwnerInfoStep.doYouOwn25percent')} ${companyName}?`}
+                    defaultValue={isUserUBO}
+                    onSelectedValue={handleNextUBOSubstep}
+                />
+            )}
+
+            {currentUBOSubstep === SUBSTEP.IS_ANYONE_ELSE_UBO && (
+                <BeneficialOwnerCheckUBO
+                    title={`${translate('beneficialOwnerInfoStep.doAnyIndividualOwn25percent')} ${companyName}?`}
```

Comment:
Maybe text can be `Do any other individual` whenever this question is asked again

Reply:
Use designs as a reference when it comes to texts. I dont have the authority to change the texts we display